code/src/components/product-card/ProductCard.test.tsx

```diff
@@ -25,7 +25,9 @@ describe("ProductCard component", () => {
   });
 
   test("should render product price", () => {
-    const productPrice = screen.getByText(mockProduct.price);
+    const productPrice = screen.getByText(
+      new RegExp(`${mockProduct.price}\\$`)
```

Comment:
why do you decided to use `RegExp` here ?
It can be easier like ```(`${mockProduct.price}$`)```

Reply:
I hope this solution makes os independent from currency, that we use. It is generally better to prefer regular expression in tests to avoid tight connection between translation of text content and element we want to test. Remember, comma or dollar should not break our tests)

Reply:
I hope this solution makes os independent from currency, that we use. It is generally better to prefer regular expression in tests to avoid tight connection between translation of text content and element we want to test. Remember, comma or dollar should not break our tests)

Reply:
Yeah. Also in the future, I will replace it with the custom utility function, which will convert numbers into currency